content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -205,9 +205,9 @@ This ensures that even pods that aren't selected by any other NetworkPolicy will
 
 ## SCTP support
 
-{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
```

Comment:
I'm wondering why this is part of this PR. What was the reason for it?

Reply:
@saschagrunert oh weird, looks like I messed up the rebase. None of these SCTP changes are mine

Reply:
Should be fixed now :+1: